manager/latest.yml

```diff
@@ -1,9 +1,9 @@
-version: 1.7.0
+version: 1.7.1
 files:
   - url: Outline-Manager.exe
-    sha512: jwXj2xZdsc4pHy9qV6gwrjqDiOwK4todnGrQbR07n9+T42XGtmDv42A3k7s4q2DTgL7NI4cgLtbdGMaqpyK8aw==
-    size: 86051640
+    sha512: fQpFkAByAz9WPO8yiEvU/UAk5CgZXQTg6/9GOVCQG3fdlKBZJf1YiKKkIZv1YbfJgenABVKwdjBj8kxmLRZhFw==
+    size: 53592400
```

Comment:
Why is the Window binary going from 86MG to 53MB? No major changes on the other platforms.
Something is off. Is the old binary working? Is the new binary working?

Reply:
@alalamav how did you build the Windows binaries? Any ideas why the size changed so much?

Reply:
After offline review, this binary seems okay